image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -170,7 +170,9 @@ defaults:
     enablePodSecurityPolicies: [< .EnablePodSecurityPolicies >]
     [<- end >]
 
-  enableOpenShiftMonitoring: false
+monitoring:
```

Comment:
How come the indentation changed here?

Reply:
Probably a mistake, now I wonder why it worked or maybe it does not work and that's why I need to add nil checks 🤔